### DISCUSSION TITLE
Triple boot option?

### DISCUSSION THREAD

**Discussion by elogh** [stefanrueger/urboot#41] [Ideas]
Hello!
In one of my projects, I'm installing a Pro Mini board with an attached HC-05 in an astro-modified camera to record image series. It would be very useful if I could install updates via the Bluetooth adapter so that I don't have to disassemble the camera too often. However, I have very little free space in the camera and no more free data lines, so dual boot via SPI flash memory is not an option.
I therefore wondered whether an over-the-air update without SPI flash would be possible if the software uses less than half of the internal flash memory. One could then save an update in the upper half of the flash with pgm_write_page() and cause urboot with a watchdog reset to install this update.
Is this idea easy to realise, or am I missing something important?

Greetings and many thanks for this great peace of software!
Erwin

**Comment by stefanrueger** [comment]
See also Issue #33

**Reply by elogh** [reply]
Ah okay, thanks.
